packages/node_modules/@cerebral/http/README.md

```diff
@@ -338,7 +338,7 @@ function someDeleteAction ({http, props}) {
     name: 'filename.png', // Default to "files"
     data: {}, // Additional form data
     headers: {},
-    onProgress(progress) {} // Upload progress
+    onProgress: 'some.signal.path' // Upload progress
```

Comment:
There is really no reason to trigger anything else than a signal :) This was method thing was just some artefact from previous implementations